### PR TITLE
[IA-549] Modify zendesk remote ff

### PR DESCRIPTION
**Pull request by debiff**
This PR modify the definition and the value of the zendesk remote FF.
| Before  | After |
| ------------- | ------------- |
| boolean  | "none" or "zendesk" or "instabug" or "web"  |

### CODE REVIEW

File: status/backend.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
       "program_active": true
     },
     "zendesk": {
```
Comment (Undermaken) [inline status/backend.json:16]: I guess `assistanceTool` should replace `zendesk`

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: status/backend.json (hunk 1 above)
Comment (debiff) [reply status/backend.json:16]: Yes, you're right.
Modified in [5c12c7f](https://github.com/pagopa/io-services-metadata/pull/525/commits/5c12c7f632ae46ef9c99cffd1949428c314a7dfc)

File: definitions.yml [hunk 2]
```diff
@@ -412,13 +412,14 @@ definitions:
       program_active:
         description: "If true, the cashback program is active and all the related in app functionalities are enabled"
         type: boolean
-  ZendeskConfig:
+  AssistanceToolConfig:
     type: object
     description: A specific config for Zendesk
     required:
-      - active
+      - tools
```
Comment (Undermaken) [inline definitions.yml:419]:
```suggestion
      - tool
```
It's more appropriate for a single value

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: definitions.yml (hunk 2 above)
Comment (Undermaken) [reply definitions.yml:419]: We should also update the `description`

`description: A specific config for Zendesk`

**Review by Undermaken — APPROVED** [review]

**Pull request merged**